app/lib/SentimentAnalyser.ts

```diff
@@ -0,0 +1,66 @@
+// @ts-ignore
+import natural, { WordTokenizer, SentimentAnalyzer } from 'natural';
+import aposToLexForm from 'apos-to-lex-form';
+import SpellCorrector from 'spelling-corrector';
+import stopword from 'stopword';
+import { pipe } from 'ramda';
+
+export class SentimentAnalyser {
+  private tokenizer: WordTokenizer;
+  private analyzer: SentimentAnalyzer;
+  private corrector: any;
+
+  constructor() {
+    // @ts-ignore
+    const { WordTokenizer, PorterStemmer, SentimentAnalyzer } = natural;
+    this.tokenizer = new WordTokenizer();
+    this.analyzer = new SentimentAnalyzer('English', PorterStemmer, 'afinn');
```

Comment:
create config variable for both
- language - default `English`
- vocabulary - default `afinn`